data/templates/container/containers.conf.j2

```diff
@@ -170,14 +170,6 @@ default_sysctls = [
 #
 #label = true
 
-# Logging driver for the container. Available options: k8s-file and journald.
```

Comment:
Leaving the global option does not preclude per container logging configuration.